src/lib/server.js

```diff
@@ -1,219 +1,140 @@
-async function graphQL(operation, query, options) {
+import {createHttpLink} from "apollo-link-http";
+import {request, widget} from '@proca/api';
+
+async function graphQL(query, variables, options) {
```

Comment:
using proca-api in the front-end is adding 40k. Unless there is a huge functional improvement, let's stick to my ugly but simple and optimised "raw" queries